src/Xunit.Categories/DatabaseTestAttribute.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using Xunit.Sdk;
+
+namespace Xunit.Categories
+{
+    [TraitDiscoverer(IntegrationTestDiscoverer.DiscovererTypeName, DiscovererUtil.AssemblyName)]
```

Comment:
The `TraitDiscoverer` needs to be in sync with the attribute. In this case `DatabaseTestDiscoverer` otherwise the tests will not be found when run.

Reply:
Nice catch thanks for finding that.